script.js

```diff
@@ -1,4 +1,523 @@
+      myOutputValue += `<b><u>Player ${playerTurn}</b></u><br>Your cards are:`;
+      myOutputValue += displayCards(playerTurn);
+      currentTotal = currentValue(playerTurn);
+
+      myOutputValue += `<br><br>Your total card value is <b>${currentTotal}</b>`;
+      master[playerTurn].blackjackValues[0] = currentTotal;
+      if (currentTotal > 21) {
+        myOutputValue += `<br>Looks like it's a <b>bust</b> 💣 Press '<b>Submit</b>' to continue to next player...${bombImage}`;
+        master[playerTurn].blackjackValues[0] = 0;
+        gameMode = "game";
+        playerTurn += 1;
+      } else {
+        myOutputValue += `<br>Do you want to '<b>hit</b>' 💥 or '<b>stand</b>' ✋🏼?<br><i>(p.s. You must have at least 17 to stand)</i>`;
+      }
+      myOutputValue += displayOneDealerCard();
+    } else if (input == "stand") {
+      // Stand message
+      if (master[playerTurn].blackjackValues[0] <= 16) {
+        return `The minimum value to '<b>stand</b>' ✋🏼 is 17 and you have <b>${master[playerTurn].blackjackValues[0]}</b>. Please enter '<b>hit</b>' 💥.`;
+      }
+      myOutputValue += `<b><u>Player ${playerTurn}</b></u><br>`;
+      myOutputValue += `You have chosen to stand. Your Blackjack value is <b>${master[playerTurn].blackjackValues[0]}</b>${standImage}`;
+      gameMode = "game";
+      playerTurn += 1;
+    } else {
+      // Error message for not entering 'hit' or 'stand'
+      myOutputValue += `<b>Error</b>! Please enter if you would like to '<b>hit</b>' 💥 or '<b>stand</b>' ✋🏼<br><br><b><u>Player ${playerTurn}</b></u><br>Your cards are:`;
+      myOutputValue += displayCards(playerTurn);
+      myOutputValue += `<br><br>Your total card value is <b>${master[playerTurn].blackjackValues[0]}</b><br><i>(p.s. You must have at least 17 to stand)</i>`;
+    }
+  }
+  //PHASE 6: RESULTS
+  else if (gameMode == "results") {
+    // List down all the cards dealt across all players
+    for (var i = 1; i < numOfPlayers; i += 1) {
+      if (master[i].playerPoints != 0) {
+        myOutputValue += `<b><u>Player ${i}</b></u>`;
+        if (master[i].split) {
+          myOutputValue += displaySplitCards(i);
+          myOutputValue += `<br>`;
+        } else {
+          myOutputValue += ` (${master[i].blackjackValues[0]})`;
+          myOutputValue += displayCards(i);
+          myOutputValue += `<br><br>`;
+        }
+      }
+    }
+    // Displays dealer cards
+    myOutputValue += `<b><u>Dealer</b></u>`;
+    myOutputValue += ` (${master["dealer"].blackjackValues[0]})`;
+    myOutputValue += displayCards(dealer);
+    myOutputValue += `<br><br>`;
+
+    // Displays final result of all players, except dealer
+    var dealerValue = 0;
+    var currentPlayerValue = 0;
+    for (var i = 1; i < numOfPlayers; i += 1) {
+      if (master[i].split) {
+        for (var j = 0; j < 2; j += 1) {
+          myOutputValue += `Player ${i} (Hand ${j + 1}): `;
+          dealerValue = master["dealer"].blackjackValues[0];
```

Comment:
could be placed before the for loop as dealerValue will not change with each player.
calculate once vs multiple times